build.rs

```diff
@@ -1,11 +1,24 @@
 extern crate bindgen;
 
+use std::env;
 use std::path::PathBuf;
 use std::process::Command;
 
 const HEADER_PATH: &str = "include/uapi/linux/nitro_enclaves.h";
 const OUT_FILE: &str = "/driver_structs.rs";
 
+fn get_gcc_sysroot() -> String {
```

Comment:
Some comments would be really useful. We should be able why this is needed anytime we open this file.

Reply:
Added and amended.